specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -10756,6 +10815,10 @@
           },
           "description": "The resource status information."
         },
+        "assignedHost": {
+          "type": "string",
+          "description": "Resource id of the dedicated host, on which the virtual machine is allocated through automatic placement."
```

Comment:
Resource id of the dedicated host, on which the virtual machine is allocated through automatic placement when associated with  a dedicated host group that has automatic placement enabled.